client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/Footer.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useEffect, useState } from 'react';
+import {
+  Box,
+  AppFooterPortal,
+  DialogButton,
+  RouteBuilder,
+  useNavigate,
+  InlineSpinner,
+  useParams,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+
+interface FooterProps {
+  hasNext: boolean;
+  next: string | null;
+  hasPrevious: boolean;
+  previous: string | null;
+  invoiceNumber?: number;
+  loading: boolean;
+  isDirty: boolean;
+  handleSave: (onSaved: () => boolean | void) => Promise<boolean | void>;
+}
+
+export const Footer: FC<FooterProps> = ({
+  hasNext,
+  next,
+  hasPrevious,
+  previous,
+  invoiceNumber,
+  loading,
+  isDirty,
```

Comment:
Don't like having all this in the footer component, but don't know a way around it unless we do that auto saving thing...